pkg/crc/services/dns/template.go

```diff
@@ -43,20 +44,21 @@ func createDnsmasqDNSConfig(serviceConfig services.ServicePostStartConfig) error
 		AppsDomain:  serviceConfig.BundleMetadata.ClusterInfo.AppsDomain,
 		ClusterName: serviceConfig.BundleMetadata.ClusterInfo.ClusterName,
 		IP:          serviceConfig.IP,
+		InternalIP:  serviceConfig.BundleMetadata.Nodes[0].InternalIP,
 	}
 
-	dnsConfig, err := createDnsConfigFile(dnsmasqConfFileValues)
+	dnsConfig, err := createDnsConfigFile(dnsmasqConfFileValues, dnsmasqConfTemplate)
 	if err != nil {
 		return err
 	}
 
 	return serviceConfig.SSHRunner.CopyData([]byte(dnsConfig), "/var/srv/dnsmasq.conf")
 }
 
-func createDnsConfigFile(values dnsmasqConfFileValues) (string, error) {
+func createDnsConfigFile(values dnsmasqConfFileValues, tmpl string) (string, error) {
```

Comment:
This is no longer mandatory tohave this as an argument.

Reply:
@cfergeau yes but in case we again need to add the hosts file for dnsmasq.

Reply:
Then the argument can be readded again at this time ;)